### DISCUSSION TITLE
The type or namespace name 'StatusItem' does not exist in the namespace 'Terminal.Gui' (

### DISCUSSION THREAD

**Discussion by GronbergInfo** [tui-cs/Terminal.Gui#3800] [General]
Hi. i just created a basic TUI with the "TerminalGuiDesigner".
When i try to run/compile the project i get the following errors:

```
0>mainView.Designer.cs(22,30): Error CS0234 : The type or namespace name 'StatusItem' does not exist in the namespace 'Terminal.Gui' (are you missing an assembly reference?)
0>mainView.Designer.cs(24,30): Error CS0234 : The type or namespace name 'StatusItem' does not exist in the namespace 'Terminal.Gui' (are you missing an assembly reference?)
0>------- Finished building project: squidDash. Succeeded: False. Errors: 2. Warnings: 0
```

Checking nuget, i already have Termina.GUI installed in the project.
Any ideas?

**Comment by GronbergInfo** [comment]
On a chance, i re-ran: _dotnet add package Terminal.Gui_
And now it compiled without errors, wierdly is "removed it self" (or by something)

**Reply by tznind** [reply]
Hi, are you targeting v1 or v2 of Terminal.Gui?

What version is listed in your csproj file?

V1 is stable and should not have compilation issues.  

V2 is in alpha, so has breaking changes.  This means you have to carefully match the build versions you are targeting to the version of TGD you have installed.

If you are able to share the source code or the commands you ran to install/setup project that would be helpful.
